.nycrc

```diff
@@ -9,7 +9,7 @@
   "reporter": [
     "html",
     "text",
-    "lcov"
+    "cobertura"
```

Comment:
I'm hoping this gives us better partial coverage (_I noticed C++ reports branch level coverage_).